scripts/populate_mock_db.py

```
@@ -0,0 +1,67 @@
+
+    # Ensure directory exists
+    os.makedirs(os.path.dirname(db_path), exist_ok=True)
+
+    conn = init_db(db_path)
+    c = conn.cursor()
+
+    # Add an appliance
+    c.execute("INSERT INTO appliance (name) VALUES (?)", ("P22P6",))
+    appliance_id = c.lastrowid
+
+    now = datetime.now()
```

Ruff (0.15.0)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)